test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -418,7 +418,7 @@ def setUp(self):
             "exclude_hosts"   : [],
             "exclude_vntypes" : [],
             "soft_limit"      : false,
-            "default"         : "256MB",
+            "default"         : "96MB",
```

Comment:
Why the change between 96mb and 256mb?  Both seem rather low for any program these days.